PyViCare/PyViCareRadiatorActuator.py

```diff
@@ -14,12 +13,27 @@ def getBatteryLevel(self) -> int:
 
     @handleNotSupported
     def getTemperature(self):
-        return self.service.getProperty("device.sensors.temperature")["properties"]["value"]["value"]
+        # Zigbee TRV reports temperature under trv.temperature, not device.sensors.temperature
+        return self.service.getProperty("trv.temperature")["properties"]["value"]["value"]
```

Comment:
No, the current temperature is measured by the sensor, the `trv.temperature` gives the target temperature that is set on the device.